source/statements.tex

```diff
@@ -1081,7 +1081,8 @@
 conversion\iref{expr.type.conv} as its leftmost subexpression can be
 indistinguishable from a \grammarterm{declaration} where the first
 \grammarterm{declarator} starts with a \tcode{(}. In those cases the
-\grammarterm{statement} is a \grammarterm{declaration}.
+\grammarterm{statement} is a declaration,
```

Comment:
One cannot say that a grammar production  rule *is* another grammar production rule. Hence, this was an attempt to improve the wording, taking advantage of the fact that  "declaration" is loosely used as a term, so it's possible to alter/embellish/clarify it's definition. This change also follows existing precedence.

Reply:
We have ambiguous grammar, and this sentence says whether you take one or the other choice. Since grammar is presented as non-terminals expanding to other non-terminals, it seems plausible to say "is a declaration" (i.e. is this branch).

Reply:
Ok to say "is considered to be a _declaration_"? That way we're claiming it's a rule rather than saying it _is_ a rule, and we've used that wording elsewhere so it has precedence. 

I'll revert the change and add "considered to be" in a separate commit. Note that it's in a separate commit so the editor can choose to apply it (or not) at his discretion.